game-core/src/main/java/games/strategy/engine/lobby/client/ui/LobbyGamePanel.java

```diff
@@ -194,20 +197,27 @@ private void mouseOnGamesList(final MouseEvent e) {
   }
 
   private static Action getHostSupportInfoAction(final GameDescription description) {
-    final String supportEmail = description == null ? "" : Strings.nullToEmpty(description.getBotSupportEmail());
-    if (supportEmail.isEmpty()) {
-      return null;
-    }
-    final String text = "Support Email for this Host is as follows. "
-        + "\n(Please copy the email address below and manually email them ONLY if something is seriously "
-        + "\nwrong with the host, like it needs to be restarted because it is down and not working at all.) "
-        + "\n\nEmail: \n" + supportEmail;
-    return SwingAction.of("Show Host Support Information/Email", e -> {
-      final JTextPane textPane = new JTextPane();
-      textPane.setEditable(false);
-      textPane.setText(text);
-      JOptionPane.showMessageDialog(null, textPane, "Host Support Info", JOptionPane.INFORMATION_MESSAGE);
-    });
+    return getBotSupportEmail(description)
+        .map(supportEmail -> {
+          final String text = "Support Email for this Host is as follows. "
+              + "\n(Please copy the email address below and manually email them ONLY if something is seriously "
+              + "\nwrong with the host, like it needs to be restarted because it is down and not working at all.) "
+              + "\n\nEmail: \n" + supportEmail;
+          return SwingAction.of("Show Host Support Information/Email", e -> {
+            final JTextPane textPane = new JTextPane();
+            textPane.setEditable(false);
+            textPane.setText(text);
+            JOptionPane.showMessageDialog(null, textPane, "Host Support Info", JOptionPane.INFORMATION_MESSAGE);
+          });
+        })
+        .orElse(null);
+  }
+
+  @VisibleForTesting
+  static Optional<String> getBotSupportEmail(final @Nullable GameDescription gameDescription) {
+    return (gameDescription != null)
```

Comment:
The nullity of `gameDescription` should already be known, bit of a code smell to have arg nullity checks on private methods. Would it be reasonable to push the nullity check up the stack? this would help when reasoning about data state, it remove the whole null case and the code should be easier to follow.